doc/api/next_api_changes/removals/22081-AL.rst

```diff
@@ -0,0 +1,11 @@
+`colorbar`` defaults to stealing space from the mappable's axes rather than the current axes
```

Comment:
```suggestion
colorbar defaults to stealing space from the mappable's axes rather than the current axes
```

This was failing the doc build, but I also think @QuLogic prefers we dont' add styling in the headlines?

Reply:
Thanks for catching that.

Reply:
Styling is okay, generally, but not cross-references.